frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/common/ErrorMessages.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Siemens AG, 2013-2016. Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package com.siemens.sw360.portal.common;
+
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Defines possible error messages that can be displayed in liferay
+ */
+public class ErrorMessages {
+
+    public static final String PROJECT_NOT_ADDED = "Project could not be added.";
+    public static final String COMPONENT_NOT_ADDED = "Component could not be added.";
+    public static final String RELEASE_NOT_ADDED = "Release could not be added.";
+    public static final String ERROR_GETTING_PROJECT = "Error fetching project from backend.";
+    public static final String ERROR_GETTING_COMPONENT = "Error fetching component from backend.";
+    public static final String ERROR_GETTING_RELEASE = "Error fetching release from backend.";
+    public static final String LICENSE_USED_BY_RELEASE =  "Request could not be processed, as license is used by at least one release!";
+    public static final String DOCUMENT_USED_BY_PROJECT_OR_RELEASE = "Document could not be processed, as it is used by other Projects or Releases!";
```

Comment:
I do not think renaming or updating can be a problem in the future as we reference documents by id. This message is only used when an attempt to delete a document returns status `IN_USE`.
I mean, this is a minor issue, obviously. I just thought one can be more precise in the wording.